manager/orchestrator/replicated/task_reaper_test.go

```diff
@@ -512,9 +514,11 @@ func TestServiceRemoveDeadTasks(t *testing.T) {
 
 	// Set both tasks to COMPLETED.
 	updatedTask3 := observedTask1.Copy()
+	updatedTask3.DesiredState = api.TaskStateCompleted
```

Comment:
I'm in favor of setting this to `api.TaskStateShutdown` because this is what happens in actual behavior (even though the test would work fine with this).